### PR TITLE
Minor update to my XD75 keymap

### PR DESCRIPTION
**Pull request by DavidRambo**
## Description

I put the finishing touches on the "PC" layer and its associated "NAVPC" layer. The thumb backspace key now functions as Ctrl when held. "NAVPC" has full-word deletion with the "C_BSPC = LCTL(KC_BSPC)" alias.  

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/xd75/keymaps/davidrambo/keymap.c [hunk 1]
```diff
@@ -72,7 +72,7 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
    NAVPC  , _______, _______, _______, _______, _______, _______, KC_MPLY, _______, _______, _______, _______, _______, _______, _______,
    _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______, _______,
-   _______, KC_LCTL, KC_LGUI, _______, KC_LCTL, _______, _______, _______, _______, _______, _______, KC_RCTL, KC_RALT, KC_RGUI, _______
+   _______, KC_LCTL, KC_LGUI, _______, KC_LCTL, CBSPC  , _______, _______, _______, _______, _______, KC_RCTL, KC_RALT, KC_RGUI, _______
```
Comment (drashna) [inline keyboards/xd75/keymaps/davidrambo/keymap.c]: Shouldn't this be: 
```suggestion
   _______, KC_LCTL, KC_LGUI, _______, KC_LCTL, C_BSPC, _______, _______, _______, _______, _______, KC_RCTL, KC_RALT, KC_RGUI, _______
```

Comment (noroadsleft) [reply keyboards/xd75/keymaps/davidrambo/keymap.c]: It's for a different macro.

https://github.com/qmk/qmk_firmware/blob/66533ee093d23f07ab93824ecd1dfcb9d763f681/keyboards/xd75/keymaps/davidrambo/keymap.c#L5 vs https://github.com/qmk/qmk_firmware/blob/66533ee093d23f07ab93824ecd1dfcb9d763f681/keyboards/xd75/keymaps/davidrambo/keymap.c#L12

Comment (drashna) [reply keyboards/xd75/keymaps/davidrambo/keymap.c]: Ah, okay. Didn't check.

Well, that brings up another issue:  Those are pretty similarly named, and may be an issue for you, in the future. 


If you're okay with that, then by all means, and let us know.

Comment (DavidRambo) [reply keyboards/xd75/keymaps/davidrambo/keymap.c]: Hey, thanks for the comments. I changed the alias for "LCTL_T(KC_BSPC)" from "CBSPC" to "BS_CTL", which I think is an improvement in clarity. I echoed this change for the bottom layer's Mac OS X version, which now has the alias of "BS_GUI".

Comment (drashna) [reply keyboards/xd75/keymaps/davidrambo/keymap.c]: Glad to have helped!  And looks good!

**Comment by drashna** [comment]
Thanks!

**Pull request merged**